packages/@glimmer/debug/lib/debug.ts

```diff
@@ -354,7 +358,7 @@ export function debugOp(program: Program, op: OpSnapshot, meta: BlockMetadata |
   if (LOCAL_DEBUG) {
     let metadata = opcodeMetadata(op.type);
 
-    let out: Dict<RawDisassembledOperand> = Object.create(null);
+    let out = Object.create(null) as Dict<RawDisassembledOperand>;
```

Comment:
This is a common change: there are a number of places where we were using `const x: Y = z` as a type
assertion, but the safety lint rules (correctly) treat that as an `any` assignment. While this is
just as unsafe, an `as` cast more idiomatically communicates an unsafe cast.

An `:` assignment *normally* represents a safe assignment, but it was only allowed *because* the
expression was `any`. I think this is a nice improvement to clarity.